sb3_contrib/common/wrappers/action_masker.py

```diff
@@ -0,0 +1,29 @@
+from typing import Callable, Union
+
+import gym
+import numpy as np
+
+
+class ActionMasker(gym.Wrapper):
```

Comment:
instead of this wrapper, could we just require the user to provide an action mask function with a specific name? (this would simplify things, no?)

Reply:
>You mean something along the lines of hasattr(env, "valid_actions")?

yes, the same way `compute_reward()` is required for `gym.GoalEnv`, we would force the user to implement an action mask function. It is a minimal requirement that should simplify a lot the code.

>but I still expect many users may need to use this wrapper on existing environments in order to provide the expected method.

we could document it (as it is only 5 lines of code if you remove the existence check) but you are right, we could just leave it without using it directly.